packages/std/src/imports.rs

```diff
@@ -34,7 +34,7 @@ extern "C" {
     #[cfg(feature = "iterator")]
     fn scan_db(start: *const c_void, end: *const c_void, order: i32) -> i32;
     #[cfg(feature = "iterator")]
-    fn next_db(key: *mut c_void, value: *mut c_void) -> i32;
+    fn next_db(counter: i32, key: *mut c_void, value: *mut c_void) -> i32;
```

Comment:
Can we call this "(iterator) ID"? Most of the code does not need to worry how it is implemented. Implementation via an incrementing number makes sense.

Reply:
Renamed.